### DISCUSSION TITLE
【1367550】Kingdom Rush Vengeance - Tower Defense

### DISCUSSION THREAD

**Discussion by Vocllum** [pjy612/SteamManifestCache#272] [Share]
Kingdom Rush Vengeance - Tower Defense
'1367550'
keys:

```
1367552;6dd002ff1dbc19423c66d76b1bcf01fa35234ab22f5e041322d8ede40b5de471
```

自费的，不知道库里有没有）
code: 525467BCBCD8CBF9
email: 1253689940@qq.com

**Comment by pjy612** [comment]
先不说库里有没有。
这些密钥都是工坊密钥。对入库用户来说没用。。。
请看一下版内顶置贴

**Reply by Vocllum** [reply]
抱歉，已更新，但是其他三款的depots我在文件内搜索不到，只有复仇有，四款我都在本地下载了的。

**Reply by pjy612** [reply]
有必要你可以校验文件完整性之后 再看看 密钥在不在。
